### PR TITLE
Update elasticsearch.rb for new version of 6.0.2 gem

### PR DESCRIPTION
**Pull request by lokanadhamm**
adding missing header parameter for new es gem

### CODE REVIEW

**Comment by stephenprater** [comment]
@lokanadhamm Thanks for the PR - do you know if there are tests for this situation?  I'd like to stay compatible with both the 5.x series of the gem and 6.x..

File: lib/newrelic/elasticsearch.rb [hunk 1]
```diff
@@ -19,19 +19,19 @@
...
+          statement = { body: body, params: params, headers: headers }
           statement[:scope] = resolver.scope
           statement[:additional_parameters] = resolver.operands
 
           NewRelic::Agent::Datastores.notice_statement(statement.inspect, elapsed) if statement
         end
 
         NewRelic::Agent::Datastores.wrap('Elasticsearch', resolver.operation_name, resolver.index, callback) do
-          perform_request_without_new_relic(method, path, params, body)
+          perform_request_without_new_relic(method, path, params, body, headers)
```
Comment (Goltergaul) [inline lib/newrelic/elasticsearch.rb:34]: whis will probably not be backwards compatible since the older version does not have a headers param

Comment (Goltergaul) [reply lib/newrelic/elasticsearch.rb:34]: FYI the headers param is introduced with version 6.0.2 of the elasticsearch gem

**Comment by dbackeus** [comment]
I get that it would be nice to have backwards compatibility with 5.x but isn't forward compatibility a bigger issue by now? It's been more than half a year since 6.0.2 was released.

**Comment by Goltergaul** [comment]
> I get that it would be nice to have backwards compatibility with 5.x but isn't forward compatibility a bigger issue by now? It's been more than half a year since 6.0.2 was released.

Yeah something should be released. Maybe release a pr with a version constraint on elasticsearch so that it cant be used with an older version?

**Comment by Goltergaul** [comment]
@stephenprater what do you think about having a version constraint and having newer gem versions onyl support elasticsearch >= 6.0.2?

**Comment by lokanadham100** [comment]
Yeah, Already created pull request with that check.
https://github.com/goldstar/newrelic-elasticsearch/pull/7

**Pull request closed without merging**